targets/python37-jupyter-pytorch-tensorflow-jupyterlab/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # ==================================================================
 # module list
 # ------------------------------------------------------------------
-# python        3.7       (apt)
+# python        3.8       (apt)
```

Comment:
by the way: we could re-install python manually if 3.8 is too new for us

Reply:
Is `python3.8` still not supported?